### ISSUE TITLE
Strings in translatable fields randomly aren't getting updated when entering new data

### ISSUE THREAD

**Issue by Slackman2015** [storyblok/storyblok#126]
**The issue occurs while working with:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**I'm submitting a ...**  (check one with "x")  
- [x] bug report  
- [ ] feature request  
- [ ] question

**Current behavior:**

The issue appears when adding new values to a translatable text field. After a few tries I can say that it seems that the issue appears randomly and manifests as follows:
When marking a field as translatable and adding a new string to it, then saving the story, closing and opening the view of the block element the string disappears. Noteworthy is that this can happen randomly and when it happens the new value hasn't been saved  in the story.
It seems that the frontend isn't syncing the data correctly.
![Bildschirmfoto 2019-11-06 um 09 08 46](https://user-images.githubusercontent.com/12159917/68297362-7efe7380-0075-11ea-9027-4d8a5efe62f0.png)
The JSON is showing another value in the field that the frontend.
![Bildschirmfoto 2019-11-06 um 09 10 49](https://user-images.githubusercontent.com/12159917/68297415-a2292300-0075-11ea-8d46-46a9fb07694b.png)

A short video illustrates where it happens and how it can be reproduced. It doesn't work always so I can't tell which variable is important for the issue.
[Bildschirmvideo aufnehmen 2019-11-06 um 08.24.42.zip](https://github.com/storyblok/storyblok/files/3814338/Bildschirmvideo.aufnehmen.2019-11-06.um.08.24.42.zip)

Sometimes when the story has been saved by the autosave app the message appears that the saved version of the story is newer and asks for copying over or overwrite the story.

I've tested it in Chrome Version 78.0.3904.87 (Offizieller Build) (64-Bit)
 and Firefox 70.0.1 (64-bit) on MacOS.

**Expected behavior:**

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Comment by onefriendaday** [comment]
Can you try this again please?

**Comment by Slackman2015** [comment]
I tested it with Firefox and Chrome again. Now it works like expected. Thx

**Comment by Slackman2015** [comment]
The bug is still an issue. When tested it did work but it still persists:
![Bildschirmfoto 2019-11-28 um 13 54 25](https://user-images.githubusercontent.com/12159917/69823091-b250a000-11e6-11ea-880c-766d48c6e00f.png)
I filled the fields and saved the content. Then I closed the component with the triangle and opened it again. The values aren't getting read correctly sometimes and another try later it saves all values correctly.
It is hard to say on which this random behavior depends on because doing the same thing it doesn't necessarily end in the same result.

[screencast.zip](https://github.com/storyblok/storyblok/files/3902646/screencast.zip)
